src/web/app/pom.xml

```diff
@@ -1959,5 +1949,25 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>wcs10</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.geoserver.extension</groupId>
+          <artifactId>gs-wcs1_0</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>wcs11</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.geoserver</groupId>
+          <artifactId>gs-wcs1_1</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+      </dependencies>
```

Comment:
Should not be part of this PR, but the next one about WCS 1.1. I would not mind too much though, as the WCS 1.1 is coming in very soon.